includes/shortcodes.php

```diff
@@ -33,9 +33,9 @@ function edd_download_shortcode( $atts, $content = null ) {
 		'sku'			=> '',
 		'price'         => '1',
 		'direct'        => '0',
-		'text'	        => isset( $edd_options[ 'add_to_cart_text' ] )  && $edd_options[ 'add_to_cart_text' ]    != '' ? $edd_options[ 'add_to_cart_text' ] : __( 'Purchase', 'edd' ),
-		'style'         => isset( $edd_options[ 'button_style' ] ) 	 	? $edd_options[ 'button_style' ] 		: 'button',
-		'color'         => isset( $edd_options[ 'checkout_color' ] ) 	? $edd_options[ 'checkout_color' ] 		: 'blue',
+		'text'	=> edd_get_option( 'add_to_cart_text', __( 'Purchase', 'edd' ) ),
+		'style' => edd_get_option( 'button_style', 'button' ),
+		'color' => edd_get_option( 'checkout_color', 'blue' ),
```

Comment:
White spacing clean up for these lines, just make sure spaces are used between the ```key``` and ```=>```